system-tests/README.md

```diff
@@ -1,119 +1,230 @@
-## System tests
+# System tests
+
+The system tests copy a file from a provider to a consumer blob storage account.
+
+## Publish/Build Tasks
+
+> ! Important Note !
+> 
+> MVD dependencies are Eclipse DataSpaceConnector(EDC) and Registration Service. Both of these dependencies 
+> are __not__ published to any central artifactory yet, so in local development we have to use locally 
+> published dependencies.
+>
+>In order to use the correct version of each repo required by the `MVD`, you need to look in [action.yml](./.github/actions/../../../.github/actions/gradle-setup/action.yml) for the hashes of the versions of the `EDC` and the `Registration Service` that are being used by the `MVD`.
+>
+> For Example, the `Registration Service` and `EDC` repository hash can be found in the _Checkout_ steps  (in the `ref` property) of [action.yml](./.github/actions/../../../.github/actions/gradle-setup/action.yml):
+
+```yml
+    - name: Checkout EDC
+      uses: actions/checkout@v2
+      with:
+        repository: eclipse-dataspaceconnector/DataSpaceConnector
+        path: DataSpaceConnector
+        ref: 3ff940b720f44826df28e893fb31344eb6faacef
+
+    - name: Checkout Registration Service
+      uses: actions/checkout@v2
+      with:
+        repository: eclipse-dataspaceconnector/RegistrationService
+        path: RegistrationService
+        ref: 374c14bcca23ddb1dcd7476a27264510e54de7fa
+```
+
+> After you have cloned the `EDC` and `Registration Service` repos locally you should run the command to
+> `reset` to the specific hash.  
+>
+> For Example:
+
+```bash
+# EDC (in the EDC root folder)
+git reset --hard 3ff940b720f44826df28e893fb31344eb6faacef
+
+# Registration Service (in the Registration Service root folder)
+git reset --hard 374c14bcca23ddb1dcd7476a27264510e54de7fa
+```
+
+> Now you can follow the rest of the process below.  
+> Once the publications are available in _Maven Central_ this process will not be necessary
+> 
+<br />
 
-The test copy a file from provider to consumer blob storage account.
+### EDC
 
-### Building MVD project
+<br />
+
+Checkout [Eclipse DataSpaceConnector repository](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector). Next, publish `EDC` libraries to local Maven artifactory by executing the following command  from `EDC` root folder.
```

Comment:
nit: maybe we can remove this checkout step now as it is already covered in previous section, Thoughts?

Same for RegistrationService section below.

Reply:
I will update the wording, the intention here is that when we are good to go with `main` repo's it can be checked out, next stage would be working with the OSSRH hosted repos